nft/src/main.sw

```diff
@@ -10,49 +10,56 @@ use abi::NFT;
 use data_structures::MetaData;
 use errors::{AccessError, ApprovalError, InitError, InputError};
 use events::{ApprovalEvent, BurnEvent, MintEvent, OperatorEvent, TransferEvent};
-use utils::{identities_equal, sender_identity};
+use utils::sender_identity;
 
 use std::{
-    address::Address,
     assert::require,
     chain::auth::{AuthError, msg_sender},
     context::{call_frames::{contract_id, msg_asset_id}, msg_amount, this_balance},
-    contract_id::ContractId,
     hash::sha256,
-    identity::Identity,
+    identity::*,
     logging::log,
     option::*,
     result::*,
     revert::revert,
     storage::StorageMap,
+    vec::Vec,
 };
 
 storage {
-    /// Determines if there is a whitelist to mint. Only set on the
-    /// initalization of the contract
+    /// Determines if there is an allow list to call the mint function
+    /// Only set on the initalization of the contract
     access_control: bool,
 
-    /// The identity that has permission to add identities to the whitelist
+    /// Stores the identity that has permission to add identities to the whitelist
+    /// Only set on the initalization of the contract
     access_control_address: Identity,
```

Comment:
maybe just `admin: Identity` ?
In the future, we may want to expand the scope of what this Identity can do beyond managing access control.